### PR TITLE
i18n(fr): Updated `showcase.md`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- Changes or translations of Starlight docs site content

#### Description

- What does this PR change? Give us a brief description.
Updated `showcase.md` based on these 2 commits 5e5a6d86aa6554ccc776f514c2e3028ec972c8b3 & a3e00b7fdafcacdfc06d64f79f584a7a8eb2a118

### CODE REVIEW

File: docs/src/content/docs/fr/showcase.md [hunk 1]
```diff
@@ -1,20 +1,41 @@
...
 
-Starlight est déjà utilisé en production sur le web. Voyez quelques exemples ci-dessous, et publiez votre propre lien si vous avez construit votre site de documentation avec Starlight !
+:::tip[Ajoutez le vôtre !]
+Avez-vous construit un site Starlight ou un outil pour Starlight ?
+Ouvrez une PR en ajoutant un lien vers cette page !
+:::
+
+## Sites
+
+Starlight is already being used in production. These are some of the sites around the web:
```
Comment (delucis) [inline docs/src/content/docs/fr/showcase.md:13]: Looks like this line missed the translation train!

File: docs/src/content/docs/fr/showcase.md [hunk 2]
```diff
@@ -1,20 +1,41 @@
...
 - [xs-dev docs](https://xs-dev.js.org)
 - [Athena OS](https://www.athenaos.org)
 - [simple-fm](https://simple.arciniega.one)
-- [Starwoof](https://starwoof.vercel.app/) - Regardez Ben et Chris d'Astro construire ce site Starlight présentant les animaux de compagnie d'Astro en temps réel ! [Twitch stream: Astro x Starlight - BLAZING FAST ECO-FRIENDLY NON-GMO ORGANIC docs](https://www.twitch.tv/videos/1841159960)
 
 Voir [les dépôts de projets publics utilisant Starlight sur GitHub](https://github.com/withastro/starlight/network/dependents).
+
+## Plugins de la communauté
+
+Ces outils communautaires, plugins et intégrations fonctionnent avec Starlight pour étendre ses fonctionnalités.
```
Comment (delucis) [inline docs/src/content/docs/fr/showcase.md:31]: In the English “community tools, plugins, and integrations”, “community” applies to all three nouns in the list, so perhaps we need to tweak this?

```suggestion
Ces outils, plugins et intégrations communautaires fonctionnent avec Starlight pour étendre ses fonctionnalités.
```

**Review by delucis — COMMENTED** [review]
Thanks @dreyfus92! Spotted a couple of details in this one.

**Comment by dreyfus92** [comment]
Thank you so much @delucis for reviewing, I've applied your suggestions 😁

**Review by delucis — APPROVED** [review]
Thanks @dreyfus92! 🚀

**Pull request merged**